packages/fractal/config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         version: null,
     },
     components: {
-        engine: '@frctl/handlebars',
+        engine: require('@frctl/handlebars'),
```

Comment:
This is the only code change in this PR, it's because core tries to [require a string](https://github.com/frctl/fractal/blob/2341030b6c512e3483b07bdb259d9fdf76769594/packages/core/src/entities/source.js#L40) by itself and that fails since in the context of this repository `@frctl/handlebars` does not exist in the node_modules tree of `@frctl/core`. It should not matter for real uses, however, since in that case core and handlebars would always exist in the same node_modules directory.